subsys/event_manager/event_manager.c

```diff
@@ -59,11 +61,23 @@ static void event_processor_fn(struct k_work *work)
 
 		trace_event_execution(eh, true);
 		if (IS_ENABLED(CONFIG_DESKTOP_EVENT_MANAGER_SHOW_EVENTS)) {
-			printk("e: %s ", et->name);
-			if (et->print_event) {
-				et->print_event(eh);
+			if (et->log_event) {
+				const size_t buf_len = CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN;
+				char event_log_buf[buf_len];
+				int pos;
+
+				pos = et->log_event(eh, event_log_buf, buf_len);
+				if (pos < 0) {
+					event_log_buf[0] = '\0';
+				}
+				if (pos >= buf_len) {
+					event_log_buf[buf_len - 2] = '~';
```

Comment:
can `CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN` be 2 or lower?

Reply:
Added range to KConfig - from 2 to 1024